src/envpak.cpp

```diff
@@ -287,8 +287,8 @@ void __cdecl fpeErr( INT, INT);		// intercepts floating point errors, and intege
 /*---------------------------- LOCAL VARIABLES ----------------------------*/
 
 // saved by hello() for byebye()
-LOCAL void (* CDEC byebyeFcn)(int exitCode) = NULL;	// exit function address
-LOCAL char cwdSave[FILENAME_MAX] = {0};						// current directory to restore at exit
+LOCAL void (* CDEC NEAR byebyeFcn)(int exitCode) = NULL;	// exit function address
```

Comment:
I recall one of the PRs removed NEAR and FAR - is this a leftover, or will it disappear when another PR gets merged?

Reply:
It looks like it was a lingering NEAR. I did a search for NEAR and FAR and it seems like that was the last one. I assume that the merging master had a conflict that wasn't resolved properly.